### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by fletchjp** [davetcc/IoAbstraction#198] [General]
Thank you for these libraries and for somewhere to discuss them. I have been using the various libraries for some years now as part of my activities with MERG (Model Railway Electronic Group) which is UK based with members around the world. I run an Arduino Special Interest Group and have many codes using IOAbstraction and TaskManagerIO. I also use LiquidCrystalIO as well. Now that I have found this discussion site I will start to use it. Best wishes John

**Comment by davetcc** [comment]
That's great, it is always good to hear about the many ways in which people use all the libraries.
